src/components/AddSenior.tsx

```diff
@@ -214,124 +250,206 @@ const AddSenior = ({
         setConfirm(true);
         setSeniors([...seniors, newSeniorObj]);
       }
-    } 
+    }
     // check after both API calls
     if (currRes.status != 200) {
-      console.log(currRes.text().then((text) => { console.log(text); }));
+      console.log(
+        currRes.text().then((text) => {
+          console.log(text);
+        })
+      );
       setError(true);
     }
 
     setSeniorData(emptySenior);
     setSelectedStudents([]);
   };
 
+  const handleImageReplace = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const files = event.target.files;
+
+    if (!files || files.length === 0) return;
+
+    const selectedFile = files[0];
+    if (!selectedFile) return;
+    const reader = new FileReader();
+
+    reader.onload = (loadEvent: ProgressEvent<FileReader>) => {
+      const dataUrl = loadEvent.target?.result;
+      if (typeof dataUrl === "string") {
+        setCurrentImage(dataUrl);
+      }
+    };
+
+    reader.readAsDataURL(selectedFile);
+  };
   return (
     <>
-      {showAddSeniorPopUp && 
-        (
-        <div className="absolute top-0 left-0 z-50 flex h-screen w-screen flex-row items-center justify-center \
-                        text-left backdrop-blur-[2px] backdrop-brightness-75 md:w-full">
-          
-          <div className={cn("sticky flex min-h-1/4 p-10 flex-col justify-between rounded-lg bg-white", 
-                          confirm || error ? "top-[12.5%] w-2/5" : "top-[5%] sm:w-4/5 md:w-1/2")}>
-          {!confirm && !error ? (
-                <>
+      {showAddSeniorPopUp && (
+        <div
+          className="absolute bottom-0 left-0 top-10 z-50 flex w-screen flex-row items-center justify-center
+                        text-left md:w-full"
+        >
+          <div
+            className={cn(
+              "top-5% flex h-[85%] w-[60%] max-w-[495px] flex-col justify-between overflow-auto rounded-lg bg-dark-teal px-6 py-9 font-merriweather text-white",
+              confirm || error
+                ? "top-[5.5%] w-2/5"
+                : "top-[2.5%] sm:w-4/5 md:w-1/2"
+            )}
+          >
+            {!confirm && !error ? (
```

Comment:
Styling for error-handling is slightly off - we probably should remove it in later sprints
![image](https://github.com/JumboCode/TheLegacyProject2/assets/74647679/ecdf2bc3-9fe8-4e48-8a90-8472d3139d7e)